frontend/src/scenes/dashboard/newDashboardLogic.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { template } from './newDashboardLogic'
+
+describe('template function in newDashboardLogic', () => {
+    it('template works', () => {
```

Comment:
should split into two tests (if being pedantic) `ignores unused variables` and `uses identified variables`